tools/quota-monitoring-alerting/python/bigquery_schemas/dashboard_view.sql

```diff
@@ -0,0 +1,21 @@
+WITH
```

Comment:
All Source files should have a License Header. Please add one

Reply:
Done